jetstream/export_json.py

```diff
@@ -187,11 +186,9 @@ def _get_experiment_logs_as_json(
 
     # convert results to JSON
     records = [dict(row) for row in results]
-    converter = cattr.GenConverter()
-    _datetime_to_json: Callable[[datetime], str] = lambda dt: dt.isoformat()
-    converter.register_unstructure_hook(datetime, _datetime_to_json)
+    records_json = AnalysisErrors.parse_obj(records).json()
 
-    return json.dumps(converter.unstructure(records), sort_keys=True), len(records)
```

Comment:
Is there any issue with they keys not being sorted anymore?

Reply:
I don't think there should be any issues. Not familiar with how partybal generates metrics though. I'm sure I can add the sorting back in if you think it's a concern?

Reply:
As long as Experimenter doesn't depend on it (which it probably shouldn't anyway) it's fine as is

Reply:
Yea agreed, and if the display/UI side cares about ordering, that is a better place to do the sorting anyway.